apps/site/pages/docs/go-live/overview.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: '  Overview'
+sidebar_label: 'Go-live'
+---
+
+import { Callout, Tab, Tabs } from 'nextra-theme-docs'
+import { CheckboxField } from '@faststore/ui'
+
+<header>
+
+# Overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+**FastStore** is a solution to develop high-performance frontends for ecommerce websites. However, there are other core modules that make up a complete digital commerce experience. At **VTEX**, some of these core modules are:
+
+| Module                      | Description                                                                                                                             |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| **Checkout**                | Receives the shopper's information necessary to process an order.                                                                       |
+| **OrderPlaced**             | Displays a successful message right after the checkout process is completed.                                                            |
+| **Login**                   | Handles user login in a store.                                                                                                          |
+| **MyAccount**               | allows customers to manage their orders and personal data (e.g., profile info, password, addresses, and credit cards) on a single page. |
+| **VTEX Intelligent Search** | Displays previous search results based on current and previous searches.                                                                |
+
+Therefore, to provide shoppers with a full checkout and post-purchase experience, you, as a developer, must guarantee the integration between your FastStore project and VTEX solutions before going live.
+
+To ensure that you have set up all configurations for your store to run smoothly.
+copy this [store configuration checklist](/tbd) and check if you have configured each item according to your desired operation.
```

Comment:
Qual é essa lista?

Reply:
Estava pensando em uma ideia sugerida pelo Pedro de utilizar a [checklist disponível na trilha "Go Live" do Help](https://help.vtex.com/en/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/1iP90RcJvlrfQhnlxM54wo#before-you-begin) e complementá-la com itens específicos da FastStore. No entanto, considerando a [checklist já existente neste PR](https://faststore-site-git-vtex-integration-faststore.vercel.app/docs/go-live/overview#storefront-developement), pode ser redundante adicionar outra lista 🤔